### DISCUSSION TITLE
Welcome to PHONK Discussions!

### DISCUSSION THREAD

**Discussion by victordiaz** [victordiaz/PHONK#87] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Noob49565489** [comment]
hello victor
i'm not a programmer, but a few months ago i remember i "coded" a script in protocoder that allowed me to control my phone's flashlight over net 
can this be done in phonk ? thank you

**Comment by junior-boop** [comment]
Hi victor, excuse me if I speak in french, 
Sur react native on utilise les composants (component ), sur flutter se sont des widgets, moi je voudrais savoir comment créer un widget ou composant sur phonk.
Comment manipule-t-on les grids, sur Phonk, comment accéder au propriété de chaque élément. Je suis entrain de monter un applications et toutes ces question sont sur la table. Parce que la documentation est aussi faible

**Reply by junior-boop** [reply]
And the last, how can I export my project in apk

**Comment by junior-boop** [comment]
> 
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
> 

Hi Guy ! I'm Daniel from Cameroon 
And I'm use phonk for started a project but I have many issues 
The first one is to pass data from my json files into à webview.
